indico/web/rh.py

```diff
@@ -77,6 +77,9 @@ class RH:
         'endpoint': None
     }
 
+    #: Like `normalize_url_spec`, but runs after the access check succeeded
+    normalize_url_spec_late = None
```

Comment:
Not a very descriptive name, but anything else I can think of is too wordy :/

Reply:
yeah, couldn't think of anything nicer either...